### ISSUE TITLE
Softbreaks are not rendered properly

### ISSUE THREAD

**Issue by jeromesimeon** [accordproject/template-studio-v2#147]
**Describe the bug**
Markdown paragraphs with a softbreak (i.e., a new line inside a paragraph) are rendered without whitespace.

**To Reproduce**
1. Copy the following markdown into a `sample.md` file:
```
## Late Delivery and Penalty.

In case of delayed delivery except for Force Majeure cases,
"Dan" (the Seller) shall pay to "Steve" (the Buyer) for every 2 days
of delay penalty amounting to 10.5% of the total value of the Equipment
whose delivery has been delayed. Any fractional part of a days is to be
considered a full days. The total amount of penalty shall not however,
exceed 55% of the total value of the Equipment involved in late delivery.
If the delay is more than 15 days, the Buyer is entitled to terminate this Contract.
```

2. Translate it to the Slate DOM with Markus:
```
markus parse --sample sample.md --slate --output initialState.json
```

3. Use that `initialState.json` file to initialize TSv2 here https://github.com/accordproject/template-studio-v2/tree/master/src/reducers

4. missing space is indicated in the red boxes
<img width="1421" alt="Screenshot 2019-11-06 at 10 01 30 AM" src="https://user-images.githubusercontent.com/670099/68309908-29c66000-007d-11ea-9eec-9fbcf5595f0d.png">

**Expected behavior**
Markdown paragraphs with new lines inside paragraphs (soft breaks) should be rendered with the appropriate space.

**Comment by jeromesimeon** [comment]
Note: The Slate DOM translation does not seem to handle soft breaks in a way that preserves them. This might be best fixed in `markdown-transform`.

**Comment by archhhh** [comment]
Followed the steps to reproduce the problem, but seems to work fine on my end. 

![image](https://user-images.githubusercontent.com/26677794/69117548-62c9f000-0ad3-11ea-9cba-bfc24b2ce094.png)

**Comment by jolanglinais** [comment]
Ah I think we just missed closing this, it was solved in [`markdown-transform` PR #113](https://github.com/accordproject/markdown-transform/pull/113)
